### ISSUE TITLE
Editor crashes when switching to another app and coming back

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-iOS#13866]
This is coming from a user report, I wasn't able to reproduce it. I wonder if there is a developer tool to help us reproduce that one.

The crash happens with Gutenberg, but also with Aztec. I couldn't find anything relevant in Sentry.

### Expected behavior

App crashes when switching back from another app.

### Actual behavior

Should not crash.

### Steps to reproduce the behavior

- Create or edit a post.
- Switch to another app (a game or another app that uses lots of memory) .
- Switch back to wpios, 💥 crash.

User adds that if wpios is killed manually, and then reopened, it opens to the main screen, and doesn't crash.

##### Tested on WPiOS 14.5 (and also current 14.6 in TestFlight).

**Comment by hypest** [comment]
We'll make another round of attempts to try to reproduce and will close if we can't repro.

**Comment by hypest** [comment]
👋 @etoledom , nice investigation! To be sure, did you indeed experience a crash _when returning to the app_ or is that a background crash the user wouldn't notice?

**Comment by etoledom** [comment]
> did you indeed experience a crash when returning to the app or is that a background crash the user wouldn't notice?

Good question, I might have misinterpreted it.

- A crash on background would be noticed in a way that returning to the app will launch it again restarting (probably) outside the editor. (This one is the one I experienced)
- The other possibility is if the user returned to the app, saw the editor and then it crashed, leaving the user seeing the springboard (the background with the apps icons). This I did not experienced and doesn't sound much like a memory issue.

I was thinking about the first described experience from the beginning, maybe because of the mention of a possible crash by big memory usage.

If the experience was the second, I was not able to reproduce it either.
